docs/src/docs/asciidoc/parts/asciidoctorj-compatitiblity-plugin.adoc

```diff
@@ -0,0 +1,68 @@
+== Asciidoctor Compatibility Plugin
```

Comment:
typo in file name, unless `compatitiblity` is a word 😄

Reply:
Yes, I know. When I realised it, I was like, *well it is only a filename and it will be deleted in 3.x anyway* so was not too fussed.